### PR TITLE
Added key info component design guidelines

### PR DESCRIPTION
**Pull request by Blackbaud-DeniseCisneros**
Added key info component design guidelines

For some reason, I can't get the anatomy image to display and I'm not sure why. I'm pretty sure it's something simple I'm just overlooking. 

Steve or John, will one of you check me on that?

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-key-info-docs](https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-key-info-docs)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-key-info-docs](https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-key-info-docs)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-key-info-docs](https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-key-info-docs)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-key-info-docs](https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-key-info-docs)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-key-info-docs](https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-key-info-docs)

File: src/app/docs/key-info/key-info-docs.component.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't use key info to display data that isn't important summary information."
+          captionType="danger"
+          imageSource="~/assets/img/guidelines/key-info/key-info-usage-5.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use definition lists for content that doesn't need extra attention."
```
[the remark this answers is not in the document]
Comment (Blackbaud-DeniseCisneros) [reply src/app/docs/key-info/key-info-docs.component.html]: I checked in with Steve to confirm, but it turns out we can't add links within the caption property.  I did add a link for the definition list in `sky-docs-design-guideline` though.

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-key-info-docs](https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-key-info-docs)

File: src/app/docs/key-info/key-info-docs.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (blackbaud-johnly) [reply src/app/docs/key-info/key-info-docs.component.html]: OK. I thought we had one links in the captions in phase 1, but looking through the phase 1 docs, I couldn't find one, so I may have been mistaken.

File: src/app/docs/key-info/key-info-docs.component.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      <sky-docs-design-guideline
+        heading="Guidelines"
+      >
+        <ul>
+          <li><a href="https://developer.blackbaud.com/skyux/design/guidelines/call-out-info">Call out information</a></li>
+        </ul>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+  </sky-docs-design-guidelines>
```
[the remark this answers is not in the document]
Comment (blackbaud-johnly) [reply src/app/docs/key-info/key-info-docs.component.html]: Looks like we still need the JSDoc comments for the Development tab. Let me know if you need help adding that content, @Blackbaud-DeniseCisneros.

**Pull request closed without merging**